composer.json

```diff
@@ -9,11 +9,13 @@
 		"dms/phpunit-arraysubset-asserts": "0.5.0",
 		"humbug/php-scoper": "0.15.0",
 		"pelago/emogrifier": "7.0.0",
+		"php-stubs/wordpress-stubs": "^6.3",
 		"phpcompatibility/phpcompatibility-wp": "2.1.4",
 		"sirbrillig/phpcs-no-get-current-user": "1.1.0",
 		"sirbrillig/phpcs-variable-analysis": "2.11.16",
 		"squizlabs/php_codesniffer": "3.7.2",
 		"symfony/polyfill-php80": "1.16.0",
+		"vimeo/psalm": "^4.22",
```

Comment:
Should we use fixed versions for the new packages?

Reply:
For tools like this one, I don't see any reasons to prevent a minor or patch update.
At the same time, I'm fine if we want to fix the version.

Reply:
Did it here: https://github.com/Automattic/sensei/pull/7097/commits/40ef9dd14f31547074ecb99afe0fd7891bea8780

Reply:
I was just curious if you know why we're using fixed versions for dev dependencies. I'm perfectly fine with using non-fixed versions for the dev dependencies. Feel free to revert the change if you like.